data/v2.0.0/countries-of-birth.json

```diff
@@ -0,0 +1,1010 @@
+[
```

Comment:
Have we discussed the impact of all languages being in one file? It means everyone will have to download all languages, even if they will never be able to use them e.g. `ga` when completing in `cy`.

Reply:
So this is to honour the existing front-end language switching solution for lookups (they're all in one file). That may well change if there turns out to be an issue with that approach or the requirements change. As it happens we only need to support Welsh and English for lookups and so the questions for eo/ga would be defined with no list url. The largest English/Welsh lists will be around 20kb which may well be around the top end of what we would want to support.

Reply:
The Readme file mentions `ga` and `eo`. Do we have a card to track the discussion around using one file per data set?

Reply:
It can be discussed on the existing card.